components/list/test/list.test.js

```diff
@@ -92,14 +78,10 @@ describe('d2l-list', () => {
 				`);
 
 				const listItem = elem.querySelector('[key="L2-2"]');
-				const listItemLayout = elem.querySelector('[key="L2-2"]').shadowRoot.querySelector('d2l-list-item-generic-layout');
-				await focusElem(listItem);
-				await waitUntil(() => listItem.hasAttribute('_focusing'), 'List item should be focused', { timeout: 3000 });
 
-				setTimeout(() => sendKeysElem(listItemLayout, 'down', testCase.keyPress));
-				await oneEvent(listItemLayout, 'keydown');
+				await sendKeysElem(listItem, 'press', testCase.keyPress);
```

Comment:
Just realized I switched this to `press` for no particular reason, but I assume using `down` was to test that it happens specifically on `keydown`. If we use `down` with real browser interactions, do we need to explicitly let the key `up` later?